src/EnergyPlus/DXCoils.cc

```diff
@@ -17769,6 +17777,27 @@ Real64 FanSpdResidualHeat(Real64 FanSpdRto, Real64 QCoilSenHeatingLoad, Real64 T
     return (TotCap - ZnSenLoad) / ZnSenLoad;
 }
 
+Real64 FanSpdResidualHeatUsingH(Real64 FanSpdRto, Real64 QCoilSenHeatingLoad, Real64 RatedAirMassFlowRate, Real64 Tout, Real64 Tin, Real64 Win)
+{
+
+    // FUNCTION INFORMATION:
+    //       AUTHOR         Yujie Xu (yujiex)
+    //       DATE WRITTEN   Jul 2024
+    //
+    // PURPOSE OF THIS FUNCTION:
+    //       Calculates residual function (desired zone heating load - actual heating coil capacity)
+    //       This is used to modify the fan speed to adjust the coil heating capacity to match
+    //       the zone heating load. This one uses Hin and Hout difference rather than Tin and Tout difference
+    //       like in FanSpdResidualHeat
+    //
+    Real64 ZnSenLoad = QCoilSenHeatingLoad;
+    // +-100 W minimum zone load?
+    if (std::abs(ZnSenLoad) < 100.0) ZnSenLoad = sign(100.0, ZnSenLoad);
+    Real64 Wout = Win;
+    Real64 TotCap = FanSpdRto * RatedAirMassFlowRate * (PsyHFnTdbW(Tout, Wout) - PsyHFnTdbW(Tin, Win));
+    return (TotCap - ZnSenLoad) / ZnSenLoad;
```

Comment:
Do you really need to iterate on this? There's only 1 unknown.., FahSpdRto.

Reply:
I guess not. I will change it to directly calculate FanSpdRto